deploying.md

```diff
@@ -0,0 +1,113 @@
+# Deploying Philanthropy Data Commons software
+
+This document guides one through deployment for the back-end services at
+https://api.philanthropydatacommons.org and
+https://auth.philanthropydatacommons.org.
+
+The front-end at https://pilot.philanthropydatacommons.org has its own process.
+
+What gets deployed? A [docker compose yaml file][1].
+
+How does it get deployed? A [bash script running under cron][2] pulls the new,
+stops the old, and starts the new compose file.
+
+How does the deploy script know how what to deploy? It [looks inside a file][3].
+
+This setup supports both automated and manual deployments that have exactly the
+same steps with exactly the same code.
+
+[1]: https://github.com/PhilanthropyDataCommons/deploy/blob/main/compose.yml
+[2]: https://github.com/PhilanthropyDataCommons/deploy/blob/main/deploy.sh
+[3]: https://github.com/PhilanthropyDataCommons/deploy/blob/e0d24b8a19ee563938f8211bca0485a97ff1a6e9/deploy.sh#L90
+
+## The common case: deploy a newly built version of the PDC service
```

Comment:
This is probably not the common case, in the long run.  The most common case would probably be a local deployment for development purposes.  Actual production (or staging) rollouts should be fairly uncommon compared to local deployments.

Reply:
Until you mentioned it, I didn't even think of a local deployment as a deployment, but rather something like "setting up a development environment." Local development would be covered by https://github.com/PhilanthropyDataCommons/service#development. I suppose it should be made more clear that this document is focused on production deployments and add a link to that local development.

Reply:
I can remove the judgment, though, to your point. It's unneeded.

Reply:
Simplified to this:
> ## To deploy a newly built version of the PDC service to production